test/e2e/capi_test.go

```diff
@@ -187,6 +189,22 @@ var _ = Describe("Running the Cluster API E2E tests", func() {
 
 				// Unset windows specific variables
 				Expect(os.Unsetenv("WINDOWS_WORKER_MACHINE_COUNT")).To(Succeed())
+
+				cred, err := azidentity.NewDefaultAzureCredential(nil)
+				Expect(err).NotTo(HaveOccurred())
+				identityClient, err := armmsi.NewUserAssignedIdentitiesClient(getSubscriptionID(Default), cred, nil)
+				Expect(err).NotTo(HaveOccurred())
+				identityRG := "capz-ci"
```

Comment:
Should this be set as `CI_RG` in azure-dev.yaml and referenced as AzureIdentityResourceGroup in [common.go](https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/main/test/e2e/common.go#L76)?

Reply:
Yeah that's probably a little better.